tests/integration/targets/vmware_rest_vcenter_vm_scenario/runme.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+source ../init-eco.sh
+
+# Generates a string starting with 'test-' followed by 4 random lowercase characters
+TINY_PREFIX="test-$(uuidgen | tr -d '-' | cut -c1-4 | tr '[:upper:]' '[:lower:]')"
```

Comment:
```suggestion
TINY_PREFIX="test-vmware_rest-$(uuidgen | tr -d '-' | cut -c1-4 | tr '[:upper:]' '[:lower:]')"
```
Since we have 3 repos making test resources, it might be helpful to differentiate the prefix so we know where resources are coming from